pyhive/presto.py

```diff
@@ -253,7 +253,7 @@ def execute(self, operation, parameters=None):
             )
 
         # Prepare statement
-        if parameters is None:
+        if parameters is None or len(parameters) == 0:
```

Comment:
```suggestion
        if not parameters:
```
It's a bit more Python-ish way.